### PR TITLE
Finishing touches for water mask tests

### PR DESCRIPTION
**Pull request by asjohnston-asf**

### CODE REVIEW

File: tests/test_water_mask.py [hunk 1]
```diff
@@ -16,3 +17,32 @@ def test_create_water_mask(tmp_path, test_data_dir):
     assert info['bands'][0]['type'] == 'Byte'
     assert info['bands'][0]['minimum'] == 1
```
Comment (cirrusasf) [inline tests/test_water_mask.py:18]: should it be assert info['bands'][0]['minimum'] == 0 ?

Comment (asjohnston-asf) [reply tests/test_water_mask.py:18]: This particular geotiff is inland, so I'm asserting that all the pixels have a value of 1.

Comment (cirrusasf) [reply tests/test_water_mask.py:18]: OK. I got it. forget about it.

**Review by cirrusasf — APPROVED** [review]
it looks to me.

**Pull request merged**